### PR TITLE
Runtime 23.08

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

File: clanlib-glibc2.34.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
+From 393c23cce9f16d4d5097ae8196f2612f93ed99ee Mon Sep 17 00:00:00 2001
```
Comment (hfiguiere) [inline clanlib-glibc2.34.patch:1]: shouldn't this patch go into the shared-module?

Comment (bbhtt) [reply clanlib-glibc2.34.patch:1]: Both should, but the shared-module is from @Grumbel and they are inactive.

Comment (bbhtt) [reply clanlib-glibc2.34.patch:1]: Issues are also blocked https://github.com/Grumbel/shared-modules/

**Comment by bbhtt** [comment]
Runs

File: clanlib-glibc2.34.patch (hunk 1 above)
Comment (hfiguiere) [reply clanlib-glibc2.34.patch:1]: oh I didn't realize it was a different one... forget about it then.

**Pull request merged**